### PR TITLE
Minor modification

### PR DESCRIPTION
**Pull request by joinimran**
As suggested by mapalko, updated the managed section as this is for AADJ devices.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/4876

### CODE REVIEW

**Comment by jvsam** [comment]
@nenonix @JohanFreelancer9 for copy review. Thank you.

**Comment by jvsam** [comment]
Hi @mapalko, this pull request has been copy edited and is ready for your final review and approval. Please let us know if there are outstanding issues to be resolved prior to merge. Thank you.

**Comment by jvsam** [comment]
@officedocsbot assign @mapalko

**Comment by mapalko** [comment]
Key trust group policy managed is for Hybrid Azure AD Joined and Azure AD Joined. The modification should be removed from that column. The addition in the last column is good. Thanks!

**Comment by joinimran** [comment]
@mapalko - changes applied. Please review and let me know if there is any other change required. Else we will be good to make it live.

**Comment by illfated** [comment]
Please see the Github system message below:

> ### Conflicting files
> windows/security/identity-protection/hello-for-business/hello-identity-verification.md

Latest change to the file was in commit https://github.com/MicrosoftDocs/windows-itpro-docs/commit/a7f31a12890a8136749f3fb9e591ef1e67d9a799 3 days ago (2019-11-02 00:29 GMT).

**Comment by joinimran** [comment]
@illfated Is there something I can do to fix this? Any help would be appreciated.

**Comment by illfated** [comment]
There should be a button named **[Resolve conflicts]** down on the right side of this page, looking like this:

image

**That button** will take you to a Github web editor where you can manipulate an overlapping version of your PR file, 
which should contain both your version and the conflicting lines.

**Comment by jvsam** [comment]
@joinimran kindly resolved the merge conflict. Thank you.

**Comment by joinimran** [comment]
Ready to go live.

**Comment by jvsam** [comment]
@mapalko, the issues have been resolved and this is ready for merging. Thank you.

File: windows/security/identity-protection/hello-for-business/hello-identity-verification.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ The table shows the minimum requirements for each deployment. For key trust in a
...
 | N/A | Windows Server 2016 AD FS with [KB4088889 update](https://support.microsoft.com/help/4088889) (hybrid Azure AD joined clients),<br> and</br>Windows Server 2012 or later Network Device Enrollment Service (Azure AD joined) | N/A | Windows Server 2012 or later Network Device Enrollment Service |
-| Azure MFA tenant, or</br>AD FS w/Azure MFA adapter, or</br>AD FS w/3rd Party MFA Adapter| Azure MFA tenant, or</br>AD FS w/Azure MFA adapter, or</br>AD FS w/3rd Party MFA Adapter | Azure MFA tenant, or</br>AD FS w/Azure MFA adapter, or</br>AD FS w/3rd Party MFA Adapter | Azure MFA tenant, or</br>AD FS w/Azure MFA adapter, or</br>AD FS w/3rd Party MFA Adapter |
+| Azure MFA tenant, or</br>AD FS w/Azure MFA adapter, or</br>AD FS w/Azure MFA Server adapter, or</br>AD FS w/3rd Party MFA Adapter.| Azure MFA tenant, or</br>AD FS w/Azure MFA adapter, or</br>AD FS w/Azure MFA Server adapter, or</br>AD FS w/3rd Party MFA Adapter | Azure MFA tenant, or</br>AD FS w/Azure MFA adapter, or</br>AD FS w/Azure MFA Server adapter, or</br>AD FS w/3rd Party MFA Adapter | Azure MFA tenant, or</br>AD FS w/Azure MFA adapter, or</br>AD FS w/Azure MFA Server adapter, or</br>AD FS w/3rd Party MFA Adapter.
```
Comment (nenonix) [inline windows/security/identity-protection/hello-for-business/hello-identity-verification.md]:
```suggestion
| Azure MFA tenant, or</br>AD FS w/Azure MFA adapter, or</br>AD FS w/Azure MFA Server adapter, or</br>AD FS w/3rd Party MFA Adapter| Azure MFA tenant, or</br>AD FS w/Azure MFA adapter, or</br>AD FS w/Azure MFA Server adapter, or</br>AD FS w/3rd Party MFA Adapter | Azure MFA tenant, or</br>AD FS w/Azure MFA adapter, or</br>AD FS w/Azure MFA Server adapter, or</br>AD FS w/3rd Party MFA Adapter | Azure MFA tenant, or</br>AD FS w/Azure MFA adapter, or</br>AD FS w/Azure MFA Server adapter, or</br>AD FS w/3rd Party MFA Adapter|
```

**Comment by joinimran** [comment]
Ready for publishing.

**Comment by jvsam** [comment]
Hello again @Dansimp. Another PR that has been copy reviewed and approved by *mapalko*. Kindly merge, thank you.

@officedocsbot assign @Dansimp

**Comment by jvsam** [comment]
Hi @Dansimp, just following up on this PR. I hope you'll find time to merge this. @mapalko already approved the proposed changes. Thank you in advance for your assistance.

**Comment by illfated** [comment]
#sign-off

**Pull request merged**